plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -23,6 +23,799 @@ hub_tags:
+      title: "Causality Actor Process Image SHA256"
+      type: string
+      description: "Causality actor process image SHA256"
+      required: false
+    causality_actor_process_signature_status:
+      title: "Causality Actor Process Signature Status"
+      type: string
+      description: "Causality actor process signature status"
+      required: false
+    causality_actor_process_signature_vendor:
+      title: "Causality Actor Process Signature Vendor"
+      type: string
+      description: "Causality actor process signature vendor"
+      required: false
+    cluster_name:
+      title: "Cluster Name"
+      type: string
+      description: "Cluster name"
+      required: false
+    container_id:
+      title: "Container Id"
+      type: string
+      description: "Container id"
+      required: false
+    contains_featured_host:
+      title: "Contains Featured Host"
+      type: string
+      description: "Contains featured host"
+      required: false
+    contains_featured_ip:
+      title: "Contains Featured IP"
+      type: string
+      description: "Contains featured IP"
+      required: false
+    contains_featured_user:
+      title: "Contains Featured User"
+      type: string
+      description: "Contains featured user"
+      required: false
+    dns_query_name:
+      title: "DNS Query Name"
+      type: string
+      description: "DNS query name"
+      required: false
+    dst_action_country:
+      title: "Dst Action Country"
+      type: string
+      description: "Dst action country"
+      required: false
+    dst_action_external_hostname:
+      title: "Dst Action External Hostname"
+      type: string
+      description: "Dst action external hostname"
+      required: false
+    dst_action_external_port:
+      title: "Dst Action External Port"
+      type: string
+      description: "Dst action external port"
+      required: false
+    dst_agent_id:
+      title: "Dst Agent Id"
+      type: string
+      description: "Dst agent id"
+      required: false
+    dst_association_strength:
+      title: "Dst Association Strength"
+      type: string
+      description: "Dst association strength"
+      required: false
+    dst_causality_actor_process_execution_time:
+      title: "Dst Causality Actor Process Execution Time"
+      type: string
+      description: "Dst causality actor process execution time"
+      required: false
+    event_id:
+      title: "Event Id"
+      type: string
+      description: "Event id"
+      required: false
+    event_sub_type:
+      title: "Event Sub Type"
+      type: string
+      description: "Event sub type"
+      required: false
+    event_timestamp:
+      title: "Event Timestamp"
+      type: integer
+      description: "Event timestamp"
+      required: false
+    event_type:
+      title: "Event Type"
+      type: string
+      description: "Event type"
+      required: false
+    fw_app_category:
+      title: "Fw App Category"
```

Comment:
If `Fw` is an acronym, then all instances within this file need to be capitalized